articles/active-directory/conditional-access/location-condition.md

```diff
@@ -57,7 +57,7 @@ A named location has the following components:
 The number of named locations you can configure is constrained by the size of the related object in Azure AD. You can configure locations based on of the following limitations:
 
 - One named location with up to 1200 IP ranges.
-- A maximum of 90 named locations with one IP range assigned to each of them.
+- In named locations, you can configure 90 objects of named locations, each can have 1200 IP ranges
```

Comment:
Can we just say: "90 named locations can be configured, each can have12000 IP ranges." We don't talk about "objects of named locations" anywhere else, so I think it might be confusing

Reply:
Hi, I like "90 named locations can be configured, each can have12000 IP range". Thanks a lot.